plugins/cmd/chainlink-ocr3-capability/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3"
+	consensustypes "github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/reportingplugins"
+	ocr3rp "github.com/smartcontractkit/chainlink-common/pkg/loop/reportingplugins/ocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+const (
+	loggerName = "PluginOCR3Capability"
+)
+
+func main() {
+	s := loop.MustNewStartedServer(loggerName)
+	defer s.Stop()
+
+	factory := func(c *values.Map) (consensustypes.Encoder, error) {
```

Comment:
Current workflow spec proposal lists encoder name as a top level param separate from encoder config. If we want to keep it that way then let's add a separate name param here as well.